tests/tests/tier0/bluechictl-status-watch-service/test_bluechictl_status_watch_service.py

```diff
@@ -3,48 +3,43 @@
 #
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
+import logging
+import os
 from typing import Dict
 
-from bluechi_test.bluechictl import LogLevel
 from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
 from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.service import Option, Section, SimpleRemainingService
 from bluechi_test.test import BluechiTest
 
+LOGGER = logging.getLogger(__name__)
+
 NODE_FOO = "node-foo"
 
 
 def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
     node_foo = nodes[NODE_FOO]
+    service = SimpleRemainingService()
+    service.set_option(Section.Service, Option.ExecStart, "/bin/true")
 
-    service = "org.eclipse.bluechi.Agent"
-    object = "/org/eclipse/bluechi"
-    interface = "org.eclipse.bluechi.Agent"
-
-    ctrl.bluechictl.set_log_level_on_node(NODE_FOO, LogLevel.INFO.value)
-    _, output = node_foo.exec_run(
-        f"busctl  get-property {service} {object} {interface} LogLevel"
-    )
-    assert "INFO" in output
+    node_foo.install_systemd_service(service)
+    print(ctrl.bluechictl.daemon_reload_node(NODE_FOO)[1])
+    print(ctrl.bluechictl.start_unit(NODE_FOO, service.name)[1])
```

Comment:
After restoring the test files (see [this comment](https://github.com/eclipse-bluechi/bluechi/pull/865/files#r1570502722)), please remove these wrapping prints. 
```suggestion
    ctrl.bluechictl.daemon_reload_node(NODE_FOO)[1]
    ctrl.bluechictl.start_unit(NODE_FOO, service.name)[1]
```